### PR TITLE
emphasize a ticdc improvement in 6.1 release notes

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.0.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ TiDB 版本：6.1.0
 - TiFlash 支持按需触发物理数据整理（Compaction）
 - MPP 实现窗口函数框架
 - TiCDC 支持将 changelogs 以 Avro 协议输出到 Kafka
+- TiCDC 支持拆分同步链路内的大事务，从而避免 TiCDC OOM 问题
```
Comment (lilin90) [inline releases/release-6.1.0.md:20]:
```suggestion
- TiCDC 支持在数据复制过程中拆分大事务，从而避免 OOM 的问题
```

File: releases/release-6.1.0.md [hunk 2]
```diff
@@ -351,6 +352,10 @@ TiDB 版本：6.1.0
 
         - 优化 Scatter Region 为批量模式，提升 Scatter Region 过程的稳定性 [#33618](https://github.com/pingcap/tidb/issues/33618)
 
+    + TiCDC
+
+        - TiCDC 支持在同步链路内对大事务进行拆分，降低了大事务对 TiCDC 复制延迟的影响，从而避免大事务导致的 TiCDC 节点 OOM 问题 [#5280](https://github.com/pingcap/tiflow/issues/5280)
```
Comment (lilin90) [inline releases/release-6.1.0.md:357]:
```suggestion
        - TiCDC 支持在数据复制过程中对大事务进行拆分，降低了大事务对 TiCDC 复制延迟的影响，从而避免了大事务导致的 TiCDC 节点 OOM 问题 [#5280](https://github.com/pingcap/tiflow/issues/5280)
```

**Review by lilin90 — APPROVED** [review]
Rest LGTM

File: releases/release-6.1.0.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ TiDB 版本：6.1.0
 - TiFlash 支持按需触发物理数据整理（Compaction）
 - MPP 实现窗口函数框架
 - TiCDC 支持将 changelogs 以 Avro 协议输出到 Kafka
+- TiCDC 支持在数据复制过程中拆分大事务，从而避免 OOM 的问题
```
Comment (leoppro) [inline releases/release-6.1.0.md:20]:
```suggestion
- TiCDC 支持在数据复制过程中拆分大事务，降低大事务对 TiCDC 复制延迟的影响
```

**Review by leoppro — APPROVED** [review]
LGTM

**Pull request merged**